includes/gateways/stripe/helpers/classStrpAppHelper.php

```diff
@@ -0,0 +1,65 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	die( 'You are not allowed to call this page directly.' );
+}
+
+class WPBDPStrpAppHelper {
+
+	/**
+	 * @var string
+	 */
+	public static $gateway_id = 'stripe';
+
+	/**
+	 * @return string
+	 *
+	 * @psalm-return 'live'|'test'
+	 */
+	public static function active_mode() {
+		return wpbdp_get_option( 'payments-test-mode' ) ? 'test' : 'live';
+	}
+
+	/**
+	 * Add education about Stripe fees.
```

Comment:
We should be consistent with doc comments for all new methods.

Reply:
Thanks! I've gone through and added comments to every new function with more consistency.